day_7/high_level.md

```diff
@@ -0,0 +1,14 @@
+# Ceasar Cipher Notes
+To build a Ceasar Cipher
+- Store you string in an array
+- Process through each letter
+- move amount of spaces specified by user
+- push new results
+
+# Checker Board
+To build a checker board
+- Store users board size requests: height, width.
+- Divide stored number by two to account for two separate characters
+- Have board print two lines - one starting with "X" and the next starting with " "
+- Access user input for height to print half its input to account for two lines.
+- Call method using user input
```

Comment:
These are really clearly thought out steps that provide a great foundation for getting more detailed as you start typing out code.